perfkitbenchmarker/configs/option_decoders.py

```diff
@@ -83,6 +83,55 @@ def Decode(self, value, component_full_name, flag_values):
     raise NotImplementedError()
 
 
+class EnumDecoder(ConfigOptionDecoder):
+  """ Verifies that the config options value is on of the allowed set.
+
+  Passes through the value unmodified
+  """
+
+  def __init__(self, option, none_ok=False, valid_values=[], **kwargs):
```

Comment:
none_ok not required, since None can be put in valid_values to make it acceptable.